### ISSUE TITLE
How to identify active tab(s)?

### ISSUE THREAD

**Issue by orschiro** [james-fray/tab-reloader#43]
Dear @james-fray,

Would you consider adding a better way to identify on which tab(s) the extension is currently active?

Thank you! 😃👍

![image](https://user-images.githubusercontent.com/170947/53475977-9c545a00-3a71-11e9-8115-806de0466b37.png)

**Comment by raggoo** [comment]
SUGGESTION to @james-fray:

I would also like n answer to this and have a suggestion.

Add some kind of tiny overlay to the active tabs so it's easy to know which ones are active.

It's really difficult when you got like 8 windows with hundreds of tabs and it  crashes to know which tabs are active when it comes back.

Possibly add a list to the "status" under the "all jobs".

Still a great addon!

**Comment by pabuisson** [comment]
Yes, I was wondering if we could add a means to identify auto-reloading tabs. I'm using it for a pinned tab most of the time, and would appreciate a way to identify if auto-reload is enabled for the tab or not.

**Comment by james-fray** [comment]
As of the next release, the extension lists all active jobs in the popup.

**Comment by pbarquei** [comment]
I'm getting acquainted with this add-on and so far I find it very interesting. One thing though I believe would be very useful is the ability to identify, from the tab (name) itself, if the extension is active. I can see that the popup lists all active jobs, however it would be much more appealing and simple if we could just spot those by simply looking at the tabs (e.g., by adding an icon).
